setup

```diff
@@ -261,6 +261,11 @@ echo  "  * agent hostname: ${AGENT_HOSTNAME}"
 echo
 
 if [ "${REMOVE_QM_ROOTFS}" == "Y" ]; then
+    # Unmount qm binds
+    qm_mounts="$(mount | grep /qm  | cut -d" " -f3)"
```

Comment:
Could you please use findmnt tool ? It's more safer than trying to mount, pipe, cut for space etc etc. 
Maybe the following command makes what you looking for?

```
# Find all mount points under /qm using findmnt
qm_mounts=$(findmnt --target /qm --output TARGET -n)

if [ -z "$qm_mounts" ]; then
    echo "No mount points found under /qm."
fi

# Unmount each mount point
for mount in $qm_mounts; do
    umount "$mount"
done
```

Reply:
Ack